loadtest/control/simulcontroller/websocket.go

```diff
@@ -36,6 +36,12 @@ func (c *SimulController) wsEventHandler(wg *sync.WaitGroup) {
 				break
 			}
 
+			if ack, ok := ev.GetData()["should_ack"]; ok && ack.(bool) {
+				go func() {
+					c.user.PostedAck(post.Id, "success", "", "")
```

Comment:
We should log the potential error returned by `PostedAck`

Reply:
Good call. Do I need to just pass the error to the `c.status` channel if there is one?

Reply:
Yup!